src/Date.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace MLL\GraphQLScalars;
+
+use function array_merge;
+use DateTimeInterface;
+
+class Date extends DateScalar
+{
+    private const DESCRIPTION /** @lang Markdown */
+        = 'A date string with format `Y-m-d`, e.g. `2011-05-23`.';
+
+    public function __construct(array $config = [])
+    {
+        parent::__construct(
+            array_merge($config, ['description' => self::DESCRIPTION])
+        );
+    }
```

Comment:
Why not just overwrite `$description`?

Reply:
I preferred the OOP way. Should I change it?

Reply:
Yes, it is way fewer lines of code for the same result:

```suggestion
    $this->description = ...
```

Reply:
🪄